index.html

```diff
@@ -1787,21 +1804,27 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
   <pre class="idl">
     typedef (DOMString or ArrayBuffer or NDEFMessageInit) NDEFMessageSource;
 
-    [Constructor, SecureContext, Exposed=Window]
```

Comment:
Why moving `Constructor` as part of this PR?

Reply:
Had to get rid of errors to see what was caused by me or not :-) feel free to submit a patch doing this separately and I can rebase